.github/workflows/chart-release.yml

```diff
@@ -29,12 +29,14 @@ jobs:
         with:
           version: v3.9.1
           
-
       - name: Add Helm repos
         run: |
+          cd charts/dft-backend
           helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo add catenax-ng-product-edc https://catenax-ng.github.io/product-edc
+          helm dependency update
           
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.4.0
+        uses: helm/chart-releaser-action@v1.4.1
```

Comment:
Please fix the workflow trigger according to [TRG 6.01](https://eclipse-tractusx.github.io/docs/release/trg-6/trg-6-1)

The Chart Releaser Action will create the git tag according to helm chart version for you.

Reply:
Requested changes done.

Reply:
Sorry, it is not. Compare lines 1-7 in this file to the suggested workflow:

```yaml
name: Release - Helm Charts

on:
  push:
    paths:
      - 'charts/**'
    branches:
      - main

```

Your workflow will trigger on pushing git tags, but this can cause trouble, as Chart Releaser Action will create tags by itself.

Instead the workflow should react on push to a specific branch. Again, referencing [TRG 6.01](https://eclipse-tractusx.github.io/docs/release/trg-6/trg-6-1/#implementation).

Sorry if I wasn't clear enough in my first comment.